app/components/App/Sidebar.jsx

```diff
@@ -39,18 +41,18 @@ export default class Sidebar extends React.PureComponent {
     }
   }
 
-  MenuLink({ iconName, className, children, ...props }) {
+  MenuLink({ title, iconName, customLink, className, children, ...props }) {
     const classes = classNames(className, { 'link-with-icon': !!iconName })
     return (
       <Link
         className={classes}
         activeClassName="is-active"
         onClick={this.closeSideBarIfMobile}
-        title={children}
+        title={title ? title : children}
         {...props}
       >
         {iconName && <RawIcon name={iconName} />}
-        <span>{children}</span>
+        {customLink ? <React.Fragment>{children}</React.Fragment> : <span>{children}</span>}
```

Comment:
I think we can avoid the fragment here

```suggestion
        {customLink ? children : <span>{children}</span>}
```

Reply:
Fair point.